.circleci/config.yml

```diff
@@ -1,6 +1,63 @@
 version: 2.1
 orbs:
   node: circleci/node@4.2.1
+commands:
+  set-environment:
+    parameters:
+      env:
+        default: "DEV"
+        type: "string"
+    steps:
+      - run: |
+        CF_SPACE=CF_SPACE_<<parameters.env>>
```

Comment:
CF_SPACE wouldn't fit this pattern - that doesn't get defined in CircleCI but is solely based on the `$CIRCLE_BRANCH` 
ie:
HHS:main => tanf-prod
raft-tech:raft-tdp-main => tanf-dev (will be tanf-staging once we have #847 completed)
all others => tanf-dev

Reply:
I was planning to add new ENV vars to circle ci that matched this pattern so I wouldn't need to do condition checking to insert these values